src/components/login/Login.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./Login.scss";
```

Comment:
```suggestion
import styles from './Login.module.scss';
```
Consider treating the assets as a module. It will prevent the style (class) leaks through the app.